src/components/select/__internal__/select-text/select-text.component.tsx

```diff
@@ -9,16 +9,8 @@ export interface SelectTextProps {
   disabled?: boolean;
   /** Value to be displayed */
   formattedValue?: string;
-  /** Label id passed from Select component  */
-  labelId?: string;
-  /** If true, the list is displayed */
-  isOpen?: boolean;
-  /** Callback function for when the Select Textbox loses it's focus. */
-  onBlur?: (ev: React.FocusEvent<HTMLElement>) => void;
   /** Callback function for when the component is clicked. */
   onClick?: (ev: React.MouseEvent<HTMLElement>) => void;
-  /** Callback function for when the Select Textbox is focused. */
-  onFocus?: (ev: React.FocusEvent<HTMLElement>) => void;
```

Comment:
**question**: may I ask what the context was behind removing these props? Assume they aren't needed anymore?

Reply:
yep they aren't passed to the component at all (explicitly or via rest spread) so I figured remove them